docs/src/GMB.md

```diff
@@ -0,0 +1,58 @@
+# GMB
+The dataset an extract from GMB corpus which is tagged, annotated, 
+and built specifically to train the classifier to predict named entities such as name, location, etc. 
+
+GMB is a fairly large corpus with a lot of annotations.
+Unfortunately, GMB is not perfect. It is not a gold standard corpus, meaning that it’s not completely human annotated and it’s not considered 100% correct. 
+The corpus is created by using already existed annotators and then corrected by humans where needed.
+
+
+```julia
+
+julia> corp = load(GMB())
+37789-element Array{Any,1}:
+ CorpusLoaders.NerOnlyTaggedWord[NerOnlyTaggedWord("NNS", "Families"), NerOnlyTaggedWord("IN", "of"), 
+NerOnlyTaggedWord("NNS", "soldiers"), NerOnlyTaggedWord("VBN", "killed"), NerOnlyTaggedWord("IN", "in"), 
```

Comment:
These don't look like NER tags to me.
They look like POS tags

Reply:
yeah, will change it to PosOnlyTagged words.
I first try to implement it for my own project  that's why I keep it in the Provided NerOnlyTagged but forgot to change it to PosOnlyTagged 
Sorry I will take from next time

Reply:
Added PosTaggedWord